app/layout.tsx

```diff
@@ -24,6 +24,7 @@ export default function RootLayout({
   children: React.ReactNode;
 }) {
   const [isActiveBurger, setIsActiveBurger] = useState(false);
+  const production = process.env.NODE_ENV === "production";
```

Comment:
```suggestion
  const isProduction = process.env.NODE_ENV === "production";
```

should be different name due to indicating that it's a boolean